### PR TITLE
Add compat data for GlobalFetch (Fetch API)

### PR DESCRIPTION
**Pull request by maboa**
This looks like it is deprecated, so flagged it as such.

### CODE REVIEW

File: api/GlobalFetch.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          "opera": [
+            {
+              "version_added": "29"
+            },
+            {
+              "version_added": "28",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": ""
```
Comment (jpmedley) [inline api/GlobalFetch.json]: Both versions of Opera should match meaning both should say added in 29. I think MDN folks would like you to keep the flag information (that it was behind a flag in 28.) That should also be the same for both versions of Opera.

File: api/GlobalFetch.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/GlobalFetch.json]: Both flavors of Opera should be `"30"`.

File: api/GlobalFetch.json (hunk 1 above)
Comment (maboa) [reply api/GlobalFetch.json]: 👍

File: api/GlobalFetch.json (hunk 2 above)
Comment (maboa) [reply api/GlobalFetch.json]: 👍

**Comment by Elchi3** [comment]
In my opinion the whole MDN page (https://developer.mozilla.org/en-US/docs/Web/API/GlobalFetch) makes no sense anymore. It is an obsolete mixin that was never observable to web developers directly.
I think we should redirect the page to https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope and not migrate any compat data for `GlobalFetch`.

cc @teoli2003 @chrisdavidmills

**Comment by chrisdavidmills** [comment]
Yeah, makes sense to me. I thought I'd already done that ;-)

**Comment by Elchi3** [comment]
Done

**Pull request closed without merging**